packages/web/src/__tests__/UserList.spec.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { MockPayloadGenerator } from 'relay-test-utils';
+import UserListDefault from '../UserList';
+import Environment from '../relay/Environment';
+
+afterEach(cleanup);
+
+describe('<UserList />', () => {
+  it('should return most recent operation', () => {
+    render(<UserListDefault />);
+    const operation = Environment.mock.getMostRecentOperation();
+    expect(operation.fragment.node.name).toBe('UserListQuery');
```

Comment:
this is not testing much, you are just testing the mock payload is working well, not our component

Reply:
removed